examples/decomposition/plot_incremental_pca.py

```diff
@@ -33,15 +33,27 @@
 y = iris.target
 
 n_components = 2
-ipca = IncrementalPCA(n_components=n_components, batch_size=10)
-X_ipca = ipca.fit_transform(X)
+X_ipca = IncrementalPCA(n_components=n_components)
```

Comment:
I don't understand the choice of `X_ipca` as a variable name for the estimator instance. I think it should stay named `ipca`.

`X_*` variable names should be reserved for 2D arrays (original data or transformed data) in this example.